ee/fleetd-chrome/src/mockServiceWorker.js

```diff
@@ -2,7 +2,7 @@
 /* tslint:disable */
 
 /**
- * Mock Service Worker (1.2.2).
+ * Mock Service Worker (1.1.0).
```

Comment:
Our package-lock.json says we're using 1.1.0
Do we need to update our dependencies?

As a general comment, seems like our fleetd-chrome infra and release process is not mature.

Reply:
Could have been a mismatch between the globally installed version on the system where this was generated and what's in the package file. This is a test dependency so if the tests are working I think we are okay. Shouldn't hurt to update it either.